help/en/html/doc/Technical/ContinuousIntegration.shtml

```diff
@@ -19,7 +19,9 @@
       <p>"Continuous Integration" is the process of rebuilding a system every time it changes, so
       that you rapidly learn of new problems and are motivated to remove existing ones.</p>
 
-      <p>The core of our system is a series of <a href="#jenkins">Jenkins</a> jobs that routinely
+      <p>The core of our system is a series of
+      <a href="#mandatory">GitHub Actions</a>
+      <a href="#jenkins">Jenkins</a> jobs that routinely
```

Comment:
I think it would be good to add the word `and` between these two links.

Reply:
Done! Thanks for noticing that.